ui/app/models/evaluation.js

```diff
@@ -23,4 +23,6 @@ export default Model.extend({
   job: belongsTo('job'),
 
   modifyIndex: attr('number'),
+
+  waitUntil: attr('date'),
```

Comment:
Do you want this as a string and then parse it out when you need to use it? Fine if the answer is "no", just wanted to call out that date parsing on models is (unless they changed that recently) eager (which could be totally fine here).

Reply:
I've been trying to do the things the expected way until it becomes a problem. Since evaluations don't get loaded by the thousands, I think this is okay here.